examples/drupal_loadtest.rs

```diff
@@ -97,12 +97,12 @@ async fn drupal_loadtest_front_page(client: &mut GooseClient) {
             }
             Err(e) => {
                 eprintln!("failed to parse front page: {}", e);
-                client.set_failure();
+                client.set_failure(&GooseMethod::GET, "/");
```

Comment:
@LionsAd Because we're pushing statistics directly to the parent thread, we have no way to know what request we're marking as failed. The previous method for determining this was a bit of an ugly kludge anyway (and not 100% correct). This complicates writing load-tests a little as you now have to specify the Method and Path (or name) that failed -- but, now `set_failure()` and `set_success()` should actually be accurate. I think this is a worthwhile change?